### PR TITLE
fix: wrong language selected when refreshing page

### PR DESCRIPTION
**Pull request by aeschi**
- added default language to select element

https://app.asana.com/0/1206159861069468/1206591453745783/f

### CODE REVIEW

File: src/components/language-select/index.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ export function LanguageSelect() {
       </svg>
 
       <svg
-        className="pointer-events-none relative right-2 z-10 col-start-1 row-start-1 h-4 w-4 self-center justify-self-end forced-colors:hidden"
```
Comment (raphael-arce) [inline src/components/language-select/index.tsx:27]: :O was that prettier or did you do that on purpose?

**Review by raphael-arce — APPROVED** [review]
one question otherwise lgtm, thx 👍

File: src/components/language-select/index.tsx (hunk 1 above)
Comment (aeschi) [reply src/components/language-select/index.tsx:27]: it was prettier. I reorders the classNames for tailwind: https://tailwindcss.com/blog/automatic-class-sorting-with-prettier

Comment (raphael-arce) [reply src/components/language-select/index.tsx:27]: that's weird then. We need to figure out it moves this class at the beginning on your computer, while on mine it moves it at the end 👍 let's look into it tomorrow :)

**Pull request merged**